lib/sprockets/http_utils.rb

```diff
@@ -92,7 +92,8 @@ def find_q_matches(q_values, available, &matcher)
         end
       end
 
-      matches.sort_by! { |match, quality| -quality }
+      i = 0
+      matches.sort_by!{ |match, quality| [-quality, i += 1] }
```

Comment:
Can we benchmark this method with real data?

Reply:
I will